aptos-move/mvhashmap/src/types.rs

```diff
@@ -102,6 +105,17 @@ pub enum MVAggregatorsError {
     DeltaApplicationFailure,
 }
 
+impl MVAggregatorsError {
+    pub fn from_panic_or(
+        err: PanicOr<DelayedFieldsSpeculativeError>,
+    ) -> PanicOr<MVAggregatorsError> {
+        match err {
+            PanicOr::CodeInvariantError(e) => PanicOr::CodeInvariantError(e),
```

Comment:
Can use map?

Reply:
we could implement map function for PanicOr, but that doesn't exist?

Reply:
Indeed